app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(team: list[Player]) -> int:
+
+    return sum(hero.get_rating() for hero in team)
+
+
+def elves_concert(team: list[Elf]) -> None:
+    [hero.play_elf_song() for hero in team]
+
+
+def feast_of_the_dwarves(team: list[Dwarf]) -> None:
+    [hero.eat_favourite_dish() for hero in team]
```

Comment:
it is better to use for loop here, list comprehension is better to use for list creation )

Reply:
I corrected it)